engine-precompiles/src/native.rs

```diff
@@ -444,6 +459,14 @@ impl<I: IO> Precompile for ExitToEthereum<I> {
         context: &Context,
         is_static: bool,
     ) -> EvmPrecompileResult {
+        // ETH transfer input format (min size 21 bytes)
+        //  - flag (1 byte)
+        //  - eth_recipient (20 bytes)
+        // ERC20 transfer input format: max 53 bytes
+        //  - flag (1 byte)
+        //  - amount (32 bytes)
+        //  - eth_recipient (20 bytes)
+        validate_input_size(input, 21, 117)?;
```

Comment:
Why max is 117 rather than 53?

Reply:
This seems due to autosave wasn't enabled.